tests/queries/0_stateless/02496_format_datetime_in_joda_syntax.reference

```diff
@@ -0,0 +1,231 @@
+2018
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'yy');
+18
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'yyy');
+2018
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'yyyy');
+2018
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'yyyyy');
+02018
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'D');
+2
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'DD');
+02
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'DDD');
+002
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'M');
+1
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'MM');
+01
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'MMM');
+Jan
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'MMMM');
+January
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'd');
+2
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'dd');
+02
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'ddd');
+002
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'a');
+PM
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'aa');
+PM
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'aaa');
+PM
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'K');
+10
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'KK');
+10
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'KKK');
+010
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'h');
+10
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'hh');
+10
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'hhh');
+010
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'H');
+22
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'HH');
+22
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'HHH');
+022
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'k');
+22
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'kk');
+22
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'kkk');
+022
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'm');
+33
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'mm');
+33
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'mmm');
+033
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 's');
+44
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'ss');
+44
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44'), 'sss');
+044
+SELECT formatDateTimeInJodaSyntax(toDateTime('2018-01-02 22:33:44', 'UTC'), 'zzzz');
```

Comment:
Maybe include a test for short forms of the time zone?

EDIT: I just see that it is deliberately not supported - we could still include a test, just for coverage.

Reply:
it is already included 
```
SELECT formatDateTimeInJodaSyntax(toDate32('2018-01-02 22:33:44'), 'zz'); -- { serverError 48 }

```